apps/client/src/locales/messages.it.xlf

```diff
@@ -796,7 +796,7 @@
       </trans-unit>
       <trans-unit id="a5e368dd1104a015e020de33a532e53e4d97525f" datatype="html">
         <source> Last Request </source>
-        <target state="new">Ultima richiesta</target>
+        <target state="translated"> Ultima richiesta </target>
```

Comment:
Removing spaces

Reply:
uhm ok done, I was just following the original spacing

Reply:
My bad, I have just reverted it.